packages/core/src/env-set/oidc.ts

```diff
@@ -1,66 +1,104 @@
 import crypto, { generateKeyPairSync } from 'crypto';
-import { readFileSync, writeFileSync } from 'fs';
+import { existsSync, readFileSync, writeFileSync } from 'fs';
 
 import { getEnv } from '@silverhand/essentials';
 import inquirer from 'inquirer';
+import { createLocalJWKSet } from 'jose';
 import { nanoid } from 'nanoid';
 
+import { exportJWK } from '@/utils/jwks';
+
 import { appendDotEnv } from './dot-env';
 import { allYes, noInquiry } from './parameters';
+import { getEnvAsStringArray } from './utils';
```

Comment:
nit: getStringArrayFromEnv ?

Reply:
`getEnvAsStringArray` implies the env is not a `string`.
I think it's good to keep it `getEnvAsStringArray`